packages/@dcl/ecs/src/systems/crdt/index.ts

```diff
@@ -118,7 +118,7 @@ export function crdtSceneSystem({
       }
 
       if (buffer.size()) {
-        transport.send(buffer.toBinary())
+        transport.send(buffer.toCopiedBinary())
```

Comment:
Shouldn't this be part of the transport _itself_ instead of the specific ecs library?

I'm thinking that all WebWorkerTransports will face the same issue